packages/shared/src/components/squads/SquadTourCard.tsx

```diff
@@ -27,7 +27,10 @@ function SquadTourCard({
     <FlexCol className={className?.container}>
       <FlexCentered className="h-80 bg-gradient-to-l from-cabbage-90 to-cabbage-50">
         <img
-          className={classNames('w-full px-12', className?.banner)}
+          className={classNames(
+            'w-full max-w-[23.5rem] pt-14',
```

Comment:
Does this max-w scale nicely for all responsive sizes? (not sure how many we support)

Reply:
Yes, the `w-full` would allow that responsiveness in respect of the total width.